sdk/sqlmanagement/Azure.ResourceManager.Sql/api/Azure.ResourceManager.Sql.netstandard2.0.cs

```diff
@@ -10083,6 +10912,16 @@ protected override void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter write
         public static bool operator !=(Azure.ResourceManager.Sql.Models.ReplicaType left, Azure.ResourceManager.Sql.Models.ReplicaType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class ResourceGroupResourceGetLongTermRetentionManagedInstanceBackupsWithLocationOptions
```

Comment:
Odd name, is this a name for property bag? If yes, not sure whether customization code can rename it

Reply:
This a property bag, we can't rename it.